narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -305,11 +330,15 @@ function print_help {
 }
 
 function process_argument_ignore_zip_files {
+	IGNORE_ZIP_FILES_PRESISTENT_FILE="$(dirname "$(readlink /proc/$$/fd/255 2>/dev/null)")/ignore_zip_files_persistent.txt"
+	IGNORE_ZIP_FILES_EXPANDING_FILE="${LIFERAY_COMMON_DOWNLOAD_CACHE_DIR}/ignore_zip_files_expanding.txt"
+	
+	if [ ! -e "${IGNORE_ZIP_FILES_EXPANDING_FILE}" ]
+	then
+		lc_log DEBUG "Creating ${IGNORE_ZIP_FILES_EXPANDING_FILE}"
```

Comment:
Put the $VARIABLE in quotes, so if it is empty for any reason, one still can observe it.
However, we don't have a rule on this and I am not sure if it's going to be approved by the superior reviewers. It's a good practice, though.